api.js

```diff
@@ -1,11 +1,8 @@
 import axios from "axios";
-import { useState } from "react";
 
-const [error, setError] = useState(null)
-
-const getArticles = () => {
+const getArticles = (sortBy, order) => {
   return axios
-    .get("https://backend-project-cr4a.onrender.com/api/articles")
+    .get(`https://backend-project-cr4a.onrender.com/api/articles/?sort_by=${sortBy}&order=${order}`)
```

Comment:
Axios has its own way to deal with the queries by passing a query object as the second argument. It'll make it easier than keep updating the url. It'll also allow reusing this function for getting articles filtered by topic to make it DRY-er

Reply:
e.g. `axios.get("https://backend-project-cr4a.onrender.com/api/articles", { params: { topic, sort_by : sortBy, order...} })`